ui.tests/test-module/specs/prefill/customprefill.spec.js

```diff
@@ -26,7 +26,7 @@ describe('Custom Prefill Test', function () {
     const pagePath = "content/forms/af/core-components-it/samples/prefill/basic.html";
     const nameTextBox = "input[name='name']",
           dobDropdown = "input[name='dob']",
-          genderRadioButton = "input[name='gender']",
+          genderRadioButton = "input[name='radiobutton-c8c660bac8_name']",
```

Comment:
Why is this change required

Reply:
so that repeatability is supported. test case selector was written on old static behaviour